SU-HSDO/suhumsci
# Fix(STN-785):  Fix back to top contextual button

# READY FOR REVIEW

## Summary
When a back-to-top block is placed through Layout Builder, the styling is still unexpectedly rendering when editing a page with the content preview button unchecked. This causes the contextual menu to be inaccessible and for Developers to be unable to remove the block. 

This work:
The back to top class was being applied in two spots, this work removes it from the `<div>` and leaves it on the anchor only as it's supposed to be. This fixes the layout builder representation on the layout builder side of the page when editing, it can be easily located and removed with the contextual filter.

I also fixed a z-index'ing issue with the back-to-top button hiding behind the local footer on the bottom of the site.

Jira Card: https://sparkbox.atlassian.net/browse/STN-785

## Need Review By (Date)
8/20

## Urgency
medium

## Steps to Test
1. `npm run test`
2. `lando drush @sparkbox_sandbox.local cr`
3. Edit any page to add the Back to Top button:
- Pick a page, click the Layout tab
- Scroll down to Section 2, Click "add Block" in the component region.
- Use the sidebar block search, search "back"
- Click Back to Top Block under H&S Blocks Category
- Uncheck the "Display Title' checkbox and Click "Add Block"
2. The block should now be added to the component region, save the layout of the page, save button at the top of the page.
3. Verify that you can click the contextual filter on the added block and remove it if you'd like.
4. Verify that the Back to Top scroll button is unaffected on the actual page display when viewing the content.
5. Verify the same on a non-logged in version of the page.
6. Verify this works on both themes the same.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
